sgv2-graphqlapi/src/main/java/io/stargate/sgv2/graphql/schema/cqlfirst/ddl/fetchers/DdlQueryFetcher.java

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.stargate.sgv2.graphql.schema.cqlfirst.ddl.fetchers;
+
+import graphql.schema.DataFetchingEnvironment;
+import io.stargate.proto.QueryOuterClass.Query;
+import io.stargate.sgv2.common.cql.CqlStrings;
+import io.stargate.sgv2.common.cql.builder.Column;
+import io.stargate.sgv2.common.cql.builder.Column.Kind;
+import io.stargate.sgv2.common.cql.builder.Column.Order;
+import io.stargate.sgv2.common.cql.builder.ImmutableColumn;
+import io.stargate.sgv2.graphql.schema.CassandraFetcher;
+import io.stargate.sgv2.graphql.web.resources.StargateGraphqlContext;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * Base class for fetchers that execute a single DDL query, such as a CREATE KEYSPACE or DROP TABLE.
+ */
+public abstract class DdlQueryFetcher extends CassandraFetcher<Boolean> {
+
+  @Override
+  protected Boolean get(DataFetchingEnvironment environment, StargateGraphqlContext context) {
+    context.getBridge().executeQuery(buildQuery(environment, context));
+    return true;
```

Comment:
Yes, exceptions will be reported as GraphQL errors.
Otherwise there's nothing useful to return to the client, just like when you do a `CREATE TABLE` in cqlsh it doesn't print any response.